### ISSUE TITLE
I can use this embed with others editors? - Question 

### ISSUE THREAD

**Issue by opauwlo** [editor-js/embed#83]
I can use this embed tool with another editor? If yes, how?

**Comment by Abhii5496** [comment]
Yes, just set the iframe src, you can see if you console a embed block,  you will find  embed inside data.
`{
    "id": "bjMEc8FA3b",
    "type": "embed",
    "data": {
        "service": "instagram",
        "source": "https://www.instagram.com/...../p/..../",
        "embed": "https://www.instagram.com/p/C3Ru2Vq/embed", //  you can use this in iframe 
        "width": 320,
        "height": 500,
        "caption": ""
    }
}`
